src/main/java/com/smartbear/readyapi/client/result/RecipeExecutionResult.java

```diff
@@ -0,0 +1,31 @@
+package com.smartbear.readyapi.client.result;
+
+import com.smartbear.readyapi.client.model.TestStepResultReport;
+
+import java.util.List;
+
+public interface RecipeExecutionResult {
+    long getTimeTaken();
+
+    String getExecutionId();
+
+    Status getStatus();
+
+    int getResultCount();
+
+    TestStepResultReport getTestStepResult(int index);
+
+    TestStepResultReport getTestStepResult(String testStepName);
```

Comment:
When would this be used since you have another method to return list for same paramters (test step name)